webui/react/src/pages/ExperimentList.settings.ts

```diff
@@ -162,6 +162,9 @@ export const settingsConfigForProject = (id: number): SettingsConfig<ExperimentL
         literal(V1GetExperimentsRequestSortBy.STATE),
         literal(V1GetExperimentsRequestSortBy.UNSPECIFIED),
         literal(V1GetExperimentsRequestSortBy.USER),
+        literal(V1GetExperimentsRequestSortBy.CHECKPOINTSIZE),
+        literal(V1GetExperimentsRequestSortBy.CHECKPOINTCOUNT),
+        literal(V1GetExperimentsRequestSortBy.SEARCHERMETRICVAL),
```

Comment:
for list types like this, is there some way we we can use type / member of V1GetExperimentsRequestSortBy?

Reply:
i took a quick look at this and it doesn't seem to be doable -- `union` wants an explicit tuple, which means `map`-ing over the values doesn't really work

Reply:
Since we're auto-generating the code we could generate this automatically for each enum. Alternately we could define an `enum` function that makes this slightly easier to write.